modepy/modes.py

```diff
@@ -648,6 +663,41 @@ def tensor_product_basis(dims, basis_1d):
             _TensorProductBasisFunction(order, [basis_1d[i] for i in order])
             for order in gnitb(len(basis_1d), dims))
 
+
+def grad_tensor_product_basis(dims, basis_1d, grad_basis_1d):
+    """Provides derivatives for each of the basis functions generated by
+    :func:`tensor_product_basis`.
+
+    :returns: a :class:`tuple` of callables, where each one returns a
+        *dims*-dimensional :class:`tuple`, one for each derivative.
+
+    .. versionadded: 2020.2
```

Comment:
```suggestion
    .. versionadded:: 2020.2
```
Does it render properly with just one colon?

Reply:
Nope, it was just a typo!